lib/rules/media-feature-name-case/__tests__/index.js

```diff
@@ -59,9 +59,6 @@ testRule({
 			code: '@media (--VIEWPORT-MEDIUM) { }',
 			description: 'ignore css variables',
 		},
-		{
-			code: '@media; @media{}',
-		},
```

Comment:
I was checking if this would hit a specific code bath, but it didn't.
I forgot to remove it again.